types/pino/index.d.ts

```diff
@@ -601,12 +601,9 @@ declare namespace P {
     type WriteFn = (o: object) => void;
 
     interface LogDescriptor {
-        pid: number;
-        hostname: string;
         level: number;
```

Comment:
[`level` might also be a string](https://getpino.io/#/docs/help?id=log-levels-as-labels-instead-of-numbers)

Reply:
The section of the documentation you referenced points to [`useLevelLabels`](https://getpino.io/#/docs/api?id=uselevellabels-boolean-deprecated), which is deprecated and slated to be removed in the next major version, so I don't think we need to handle it in the types.